.pre-commit-config.yaml

```diff
@@ -91,10 +91,12 @@ repos:
   #         - --extra-keys
   #         - metadata.kernelspec metadata.language_info.version metadata.vscode
   - repo: https://github.com/nbQA-dev/nbQA
-    rev: 1.6.3
+    rev: 1.7.0
     hooks:
       - id: nbqa-black
       - id: nbqa-ruff
+        args:
+          - --ignore=B018
```

Comment:
This seems to do the trick; however, compared to Hans-Martin's suggestion, the rule configuration is now spread over two files.